src/utils/firebase.ts

```diff
@@ -0,0 +1,43 @@
+import firebase from 'firebase/compat/app'
+import { getMessaging, getToken, onMessage } from 'firebase/messaging'
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyDszHtJ4CJq0mwjBJ1pTt5OOzG5tiooEsg',
+  authDomain: 'test-bace2.firebaseapp.com',
+  projectId: 'test-bace2',
+  storageBucket: 'test-bace2.appspot.com',
+  messagingSenderId: '337703820408',
+  appId: '1:337703820408:web:2fb16ef71941817dec618d',
+}
+const firebaseApp = firebase.initializeApp(firebaseConfig)
+const messaging = getMessaging(firebaseApp)
+
+export const fetchToken = () => {
+  return getToken(messaging, {
+    vapidKey: 'BCH2laZcZj5fJyW2Od-iXyAy8OhJ2jpJDWJornW6JDSOi29IFeNjpIqrg-n-Ge9YExxpNUWeomKMQNFM4tx2t5Y',
```

Comment:
move firebase config to env. btw, do you check firebase security checklist? I didnt work with firebase notification before, so can you have a quick introduce it to me tomorrow?